terraform/modules/volterra/main.tf

```diff
@@ -80,6 +80,9 @@ resource "volterra_app_type" "at" {
   features {
     type = "TIMESERIES_ANOMALY_DETECTION"
   }
+  business_logic_markup_setting {
+    disable = true
```

Comment:
``enable = true`` would turn on redirect learning, I think. See the [provider docs](https://registry.terraform.io/providers/volterraedge/volterra/latest/docs/resources/volterra_app_type#business-logic-markup-setting).